bin/strata-cli/src/cmd/faucet.rs

```diff
@@ -45,47 +45,8 @@ pub async fn faucet(args: FaucetArgs, seed: Seed, settings: Settings) {
     #[cfg(feature = "strata_faucet")]
```

Comment:
I'd assume the cli message 2 lines above should also be displayed only if we passed the address verification, as this challenge is probably the POW challenge we have in the message below.

Feel free to add an alternative message in the lines of "Making a faucet request" or "Requesting BTC from the faucet" so the user gets some acknowledgment.

Reply:
Makes sense, will make the necessary changes.